### PR TITLE
ID21498: Added fix for testing

### PR DESCRIPTION
**Pull request by sviridovartem**

### CODE REVIEW

File: webapplication/tasks.py [hunk 1]
```diff
@@ -85,4 +85,6 @@ def run_publisher_k8s(ctx):
 @task
 def test(ctx):
     wait_port_is_open(os.getenv('POSTGRES_HOST', 'db'), 5432)
-    ctx.run('python -m manage test')
+    ctx.run(f'coverage run ./manage.py test --noinput')
```
Comment (adolgarev) [inline webapplication/tasks.py:88]: use python -m manage instead of ./manage.py

**Review by adolgarev — COMMENTED** [review]
Where coverage.xml is put and how is used?

File: webapplication/tasks.py (hunk 1 above)
Comment (sviridovartem) [reply webapplication/tasks.py:88]: coverage run -m manage test --noinput?

**Comment by sviridovartem** [comment]
> Where coverage.xml is put and how is used?

we put it in `webapplication` folder, after restarting tests we removing all project folder
showing coverage results in jenkins coverage report app
if we don't need such report we can remove all coverage part and show only test log in jenkins console

**Review by adolgarev — COMMENTED** [review]
Have you tried to run `coverage run -m manage test`?
Is coverage.xml somehow displayed at Jenkins?

**Review by adolgarev — APPROVED** [review]

**Pull request merged**